Failing test: cargo test -F dev

When running tests while turning on the `dev` feature while the `der` feature is off, the tests fail to compile:

```
error[E0599]: no associated function or constant named `from_der` found for struct `Signature<C>` in the current scope
   --> ecdsa/src/dev.rs:208:44
    |
208 |                 let sig = match Signature::from_der(sig) {
    |                                            ^^^^^^^^ associated function or constant not found in `Signature<_>`
...
318 |     new_wycheproof_test!(wycheproof_mock, "wycheproof-mock", MockCurve);
    |     ------------------------------------------------------------------- in this macro invocation
    |
   ::: ecdsa/src/lib.rs:185:1
    |
185 | pub struct Signature<C: EcdsaCurve> {
    | ----------------------------------- associated function or constant `from_der` not found for this struct
    |
note: if you're trying to build a new `Signature<_>` consider using one of the following associated functions:
      Signature::<C>::from_bytes
      Signature::<C>::from_slice
      Signature::<C>::from_scalars
   --> ecdsa/src/lib.rs:200:5
    |
200 |     pub fn from_bytes(bytes: &SignatureBytes<C>) -> Result<Self> {
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
...
212 |     pub fn from_slice(slice: &[u8]) -> Result<Self> {
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
...
237 |     pub fn from_scalars(r: impl Into<FieldBytes<C>>, s: impl Into<FieldBytes<C>>) -> Result<Self> {
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    = help: items from traits can only be used if the trait is implemented and in scope
    = note: the following trait defines an item `from_der`, perhaps you need to implement it:
            candidate #1: `Decode`
    = note: this error originates in the macro `new_wycheproof_test` (in Nightly builds, run with -Z macro-backtrace for more info)

For more information about this error, try `rustc --explain E0599`.
error: could not compile `ecdsa` (lib test) due to 1 previous error
warning: build failed, waiting for other jobs to finish...
```

--- a/ecdsa/Cargo.toml
+++ b/ecdsa/Cargo.toml
@@ -39,7 +39,7 @@ alloc = ["elliptic-curve/alloc", "signature/alloc", "spki/alloc"]
 std = ["alloc", "elliptic-curve/std"]
 
 algorithm = ["dep:rfc6979", "digest", "elliptic-curve/arithmetic"]
-dev = ["algorithm", "digest/dev", "elliptic-curve/dev"]
+dev = ["algorithm", "der", "digest/dev", "elliptic-curve/dev"]
 der = ["dep:der"]
 digest = ["dep:digest", "elliptic-curve/digest", "signature/digest"]
 getrandom = ["elliptic-curve/getrandom"]

